egs/wsj/s5/steps/cleanup/segment_long_utterances.sh

```diff
@@ -0,0 +1,389 @@
+#!/bin/bash
+
+# Copyright 2014  Guoguo Chen
+#           2016  Vimal Manohar
+# Apache 2.0
+
+set -e 
+set -o pipefail
+
+. path.sh
+
+set -u
+
+# Uniform segmentation options
+max_segment_duration=30
+overlap_duration=5
+
+# Decode options
+graph_opts=
+nj=4
+lmwt=10
+
+# TF-IDF similarity search options
+max_words=1000
```

Comment:
I think there should be about 1 to 3 lines of explanation in a comment for each of these configuration
options.  And I don't think it's necessary to have a config file in conf/.  Do you really believe that different
setups will need different options?  And is it realistic that a user (i.e. not you) would take the time to
understand so many options and to figure out how to tune them?
Please also see if there are any options that can be deleted now that you have more experience, e.g. those 'factor' options, if they are all left at 1, are they useful?